src/sdk/models/policies.ts

```diff
@@ -78,6 +78,12 @@ export interface PolicyAuditEntry extends IEntityIdentifier<number> {
 	recorded_on?: number;
 }
 
+export interface PolicyAction extends IEntityIdentifier<string> {
+	name?: string;
+	description?: string;
+	destructive?: boolean;
+}
```

Comment:
This looks like a duplicate of `PolicyActionDescriptor`

Reply:
Looks like PolicyActionDescriptor is missing some properties as well, might want to add exclusive, destructive, requires_existing, removes_item, and prompt to PolicyActionDescriptor after removing this.

Reply:
Yeah, they are similar currently and it might make sense to combine them. Thought it might make sense to separate them at first, i.e. the policy action details provided by the `/v1/policies/actions` endpoint and the policy action details once assigned to a tracking group. The case for keeping them separate would be that the unassigned action details might contain information that is not needed once assigned to a tracking group, e.g. the destructive flag or the exclusive flag.

Reply:
Updated in new commit.